langchain/vectorstores/opensearch_vector_search.py

```diff
@@ -428,7 +428,10 @@ def from_texts(
         opensearch_url = get_from_dict_or_env(
             kwargs, "opensearch_url", "OPENSEARCH_URL"
         )
-        client = _get_opensearch_client(opensearch_url)
+        # Remove the opensearch_url from kwargs to avoid passing
+        # the value twice for argument opensearch_url
+        kwargs.pop("opensearch_url")
+        client = _get_opensearch_client(opensearch_url, **kwargs)
```

Comment:
what about all the other kwargs below?

Reply:
Sorry, didn't get your question?

Reply:
there are a lot of other arguments in "kwargs" that we use below. do we want to pop them as well? or are they relevant for the client

Reply:
Got it. I have updated the code to pop all those arguments before getting the opensearch client.